resip/dum/SubscriptionCreator.cxx

```diff
@@ -15,6 +15,7 @@ SubscriptionCreator::SubscriptionCreator(DialogUsageManager& dum,
 
    mLastRequest->header(h_Event).value() = event; 
    mLastRequest->header(h_Expires).value() = subscriptionTime;
+   mLastRequest->header(h_Accepts).push_back( Mime( "application","pidf+xml") );
```

Comment:
Yup :)

On Thu, Aug 11, 2016 at 3:33 AM, Daniel Pocock <notifications@github.com>
wrote:

> In resip/dum/SubscriptionCreator.cxx
> <https://github.com/resiprocate/resiprocate/pull/52#discussion_r74378435>:
>
> > @@ -15,6 +15,7 @@ SubscriptionCreator::SubscriptionCreator(DialogUsageManager& dum,
> >
> >     mLastRequest->header(h_Event).value() = event;
> >     mLastRequest->header(h_Expires).value() = subscriptionTime;
> > +   mLastRequest->header(h_Accepts).push_back( Mime( "application","pidf+xml") );
>
> @MateusBellomo <https://github.com/MateusBellomo> remove the change from
> SubscriptionCreator and try using addSupportedMimeType(NOTIFY,
> Pidf::getStaticType()) on the MasterProfile in your apps/telepathy code
> in pull request #53 <https://github.com/resiprocate/resiprocate/pull/53>.
> Does that sound correct @sgodin <https://github.com/sgodin> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/resiprocate/resiprocate/pull/52/files/3e560b1c7f1fd67ba053d282961fd40561059262#r74378435>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtNUV1kSOkmVGpqZawOGsouTDfJ2WZ0ks5qetBdgaJpZM4JYfsi>
> .
>